app/src/ui/history/compare.tsx

```diff
@@ -527,9 +533,24 @@ export class CompareSidebar extends React.Component<
     this.textbox = textbox
   }
 
-  private onNotificationBannerDismissed = () => {
+  private onNotificationBannerDismissed = (reason: DismissalReason) => {
     this.props.dispatcher.setDivergingBranchBannerVisibility(false)
     this.props.dispatcher.recordDivergingBranchBannerDismissal()
+
+    if (reason === undefined) {
```

Comment:
Can `reason` ever be `undefined` here? What's the context for hitting this code path?

Reply:
No it cannot, just fixed ⚡️